lhotse/dataset/transforms/extra_padding.py

```diff
@@ -0,0 +1,40 @@
+from lhotse import CutSet
+from lhotse.utils import LOG_EPSILON
+
+
+class ExtraPadding:
```

Comment:
@danpovey we can use this one in snowfall (and when we move on to long-recording corpora we can extend it to find and use the real acoustic context)